vue-components/src/components/Dropdown.vue

```diff
@@ -159,11 +159,14 @@ export default defineComponent( {
 			// the following comment generates the event's description for the docs tab in storybook
 			/**
 			 * This event is emitted whenever an item is selected on the Dropdown.
+			 * As long as the selected item is different from the current selection.
```

Comment:
Super nitpicky, but IMO this would read better if it was part of the previous sentence, i.e. "This event is emitted whenever an item is selected on the Dropdown as long as the selected item is different from the current selection."

Reply:
replaced full stop by a coma, but kept the comment in two lines to avoid surpassing the character limit. Is that ok, or should I disable max-len?